modules/gapi/test/cpu/gapi_imgproc_tests_cpu.cpp

```diff
@@ -337,6 +337,54 @@ INSTANTIATE_TEST_CASE_P(BoundingRectVector32STestCPU, BoundingRectVector32STest,
                                  Values(IMGPROC_CPU),
                                  Values(IoUToleranceRect(1e-5).to_compare_obj())));
 
+INSTANTIATE_TEST_CASE_P(FitLine2DMatVectorTestCPU, FitLine2DMatVectorTest,
+                        Combine(Values(CV_8U, CV_8S, CV_16U, CV_16S,
+                                       CV_32S, CV_32F, CV_64F),
+                                Values(cv::Size(8, 0), cv::Size(1024, 0)),
+                                Values(-1),
+                                Values(IMGPROC_CPU),
+                                Values(AvgDiffToleranceVec<float, 4>(0.01).to_compare_obj()),
+                                Values(DIST_L1, DIST_L2, DIST_L12, DIST_FAIR,
+                                       DIST_WELSCH, DIST_HUBER)));
+
+#define INSTANTIATE_FITLINE_2DVECTOR_TEST(type) INSTANTIATE_TEST_CASE_P(                      \
```

Comment:
I would just use the default `INSTANTIATE_TEST_CASE_P` for each kernel

Reply:
OK